modules/home-manager/tmux.nix

```
@@ -0,0 +1,33 @@
+{ config, pkgs, lib, ... }:
+let
+  cfg = config.programs.tmux.catppuccin;
+
+  plugin = with builtins;
+    with pkgs;
+    let rev = "4e48b09a76829edc7b55fbb15467cf0411f07931";
+    in pkgs.tmuxPlugins.mkTmuxPlugin {
```

```suggestion
    in tmuxPlugins.mkTmuxPlugin {
```
`pkgs` is already in scope.